AuthenticationAndAuthorization/DataAccounts/ApplicationDbContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+
+namespace DataAccounts
+{
+    public class ApplicationDbContext : IdentityDbContext<User, IdentityRole<Guid>, Guid>
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
+        : base(options)
+        {
+        }
+        public DbSet<User> Accounts { get; set; }
```

Comment:
Let's name it `Users`. Because Accounts in English also stands for bank accounts, and this might confuse the other developers